android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -59,7 +60,7 @@ public TiUILabel(final TiViewProxy proxy)
 			protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec)
 			{
 				// Only allow label to exceed the size of parent when it's size behavior with both wordwrap and ellipsize disabled
-				if (!wordWrap && !ellipsize && layoutParams.optionWidth == null && !layoutParams.autoFillsWidth) {
+				if (!wordWrap && ellipsize != null && layoutParams.optionWidth == null && !layoutParams.autoFillsWidth) {
```

Comment:
It behaves the exact same way. That's why I changed this - because `ellipsize` becomes `null` when you set it to `false` in JS. Since this isn't a Boolean anymore `null` respresents what was `false` before.